src/Akka.Streams.Kafka/Stages/Consumers/Actors/KafkaConsumerActor.cs

```diff
@@ -345,6 +345,7 @@ private void ProcessResult(IImmutableSet<TopicPartition> partitionsToFetch, Cons
                 {
                     var messages = ImmutableList<ConsumeResult<K, V>>.Empty.Add(consumedMessage);
                     stageActorRef.Tell(new KafkaConsumerActorMetadata.Internal.Messages<K, V>(request.RequestId, messages));
+                    // _requests = _requests.Remove(stageActorRef);
```

Comment:
This is what I was talking about in issue #58 - will fix this in a separate PR, because need to rewrite request/response management between consuming actor and stages.